lib/rubocop/cop/chef/deprecation/cheffile.rb

```diff
@@ -22,18 +22,19 @@ module Chef
       # The Librarian-Chef depsolving project is no longer maintained and a Cheffile should not be used for cookbook depsolving. Consider using Policyfiles instead. If the Policyfiles model is not compatible with your workflow you may find Berkshelf offers a more similar, and still supported, experience to Librarian-Chef.
       #
       module ChefDeprecations
-        class Cheffile < Cop
+        class Cheffile < Base
           include RangeHelp
 
           MSG = 'The Librarian-Chef depsolving project is no longer maintained and a Cheffile should not be used for cookbook depsolving. Consider using Policyfiles instead.'
 
-          def investigate(processed_source)
-            return if processed_source.blank?
+          def on_new_investigation
+            f_name = processed_source.file_path
```

Comment:
We limit the filename in the config so there's no need to do the file name check again.

Reply:
Fixed.

Reply:
@scottvidmar This doesn't look fixed yet

Reply:
Now it should be fixed.